scripts/build_torch_wheels.sh

```diff
@@ -6,7 +6,7 @@ set -x  # Display commands being run.
 PYTHON_VERSION=$1
 RELEASE_VERSION=$2  # rX.Y or nightly
 BUILD_CPP_TESTS="${3:-0}"
-DEFAULT_PYTHON_VERSION=3.6
+DEFAULT_PYTHON_VERSION=3.8
```

Comment:
Is this supposed to be 3.9?

Reply:
Missed the PR description. So what I guess my real question is what makes this patch support python 3.9?

Reply:
The default is 3.8, and on GCB we can set python version per trigger.